### PR TITLE
feat: add Landing Page

### PR DESCRIPTION
**Pull request by sagargse**
### Added the Landing Page to the App

image

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,41 +1,17 @@
...
-            rel="noreferrer"
-          >
-            Contribute
-          </a>
-        </div>
-      </header>
+      <BrowserRouter>
+        <Routes>
+          <Route path="/" element={<LandingPage/>}></Route>
+          <Route path="/mockman" element={<Mockman />} />
```
Comment (maitrakhatri) [inline src/App.js:12]: add route for 404 page

File: src/index.css [hunk 2]
```diff
@@ -1,13 +1,11 @@
...
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
+:root{
+  --primary-color: #574DEC;
+  --text-color: black;
```
Comment (maitrakhatri) [inline src/index.css]: use hex code for colors

**Review by maitrakhatri — APPROVED** [review]
clean code.
good folder structure

**Review by Shailseen — COMMENTED** [review]
code looks clean and pages looks also nice.

**Review by Shailseen — APPROVED** [review]
codes looks good and ui looks nice

File: src/App.js (hunk 1 above)
Comment (sagargse) [reply src/App.js:12]: sure

**Comment by sagargse** [comment]
> clean code. good folder structure

Thanks!

File: src/index.css (hunk 2 above)
Comment (sagargse) [reply src/index.css]: Okay!

**Comment by sagargse** [comment]
> code looks clean and pages looks also nice.

Thanks!

**Pull request merged**